language/predefined_spec.rb

```diff
@@ -1116,9 +1118,55 @@ def obj.foo2; yield; end
                                  the value.
 TOPLEVEL_BINDING     Binding     A Binding object representing the binding at Ruby’s top level—
                                  the level where programs are initially executed.
+TRUE                 TrueClass   Synonym for true (deprecated in Ruby 3).
 =end
 
 describe "The predefined global constants" do
+  describe "TRUE" do
+    ruby_version_is "3.0" do
+      it "is no longer defined" do
+        Object.const_defined?(:TRUE).should == false
+      end
+    end
+
+    ruby_version_is "2.7"..."3.0" do
```

Comment:
Even in 2.5 it was already deprecated, so it should be `ruby_version_is ""..."3.0" do`

Reply:
Good point, thank you!